app/views/help/wiki_syntax_detailed.html.erb

```diff
@@ -259,6 +259,10 @@ tooltip of "WHO"
       <p>Include a wiki page. Example:</p>
       <pre><code>{{include(Foo)}}</code></pre>
     </dd>
+    <dt><code>child_pages</code></dt>
+    <dd>
+      <p>Displays a list of child pages. With no argument, it displays the child pages of the current wiki page.</p>
```

Comment:
> With no argument, it displays the child pages of the current wiki page

Does that mean it accepts an argument? What does the argument mean?
Or do you want to say it will never accept an argument?

I actually like the way it was done for `include`, specifying an example on how to use it.